src/wp-includes/load.php

```diff
@@ -116,7 +116,7 @@ function wp_fix_server_vars() {
  * @access private
  *
  * @global string $required_php_version The required PHP version string.
- * @global string $wp_version           The ClassicPress version string.
+ * @global string $wp_version           The WordPress version string.
```

Comment:
This should be `ClassicPress`, and we should change it to use `classicpress_version()` and remove the global variable usage.  Note that the string below says "ClassicPress vX requires".

Reply:
Ah and I was thinking to put double strings there, related to #44 :
* one that reads `@global string $wp_version           The WordPress version string.`
* additional one that reads `@global string $cp_version           The ClassicPress version string.`

Reply:
In this function, the `$wp_version` is only used in the following string:

```
'Your server is running PHP version %1$s but ClassicPress %2$s requires at least %3$s.'
```

So, this should be `classicpress_version()`, and any uses of `$wp_version` in this function should be removed.  The `@global` annotation can also be removed because the function will no longer reference a global variable for the version (it will call `classicpress_version()` instead).

Reply:
Allright, and that is what needs to be done in #44, correct?

So this line should become something like this?
`* calls classicpress_version()           The ClassicPress version string.`

Reply:
That line can just be removed. PHPDoc annotations like `@global` are only used for certain things. (I think we should ultimately get rid of them, since that information is already in the code, but that's for another day.)